code/modules/mob/living/carbon/xenomorph/abilities/ravager/ravager_powers.dm

```diff
@@ -43,6 +43,8 @@
 	for(var/mob/M as anything in mobs_in_range)
 		if(empower_targets >= max_targets)
 			break
+		if(!istype(M, /mob/living))
+			break
```

Comment:
This should still be `continue` as before here not `break`, otherwise that's gonna screw up the count

IMO just replace loop above to `for(var/mob/living/M in mobs_in_range)` instead so it's done auto

Reply:
replaced